modules/python/src2/typing_stubs_generation/api_refinement.py

```diff
@@ -192,6 +195,83 @@ def setMouseCallback(
     )
 
 
+def refine_dnn_module(root: NamespaceNode) -> None:
+    if "dnn" not in root.namespaces:
+        return
+    dnn_module = root.namespaces["dnn"]
+
+    """
+    class LayerProtocol(Protocol):
+        def __init__(
+            self, params: dict[str, DictValue],
+            blobs: typing.Sequence[cv2.typing.MatLike]
+        ) -> None: ...
+
+        def getMemoryShapes(
+            self, inputs: typing.Sequence[int]
+        ) -> typing.Sequence[int]: ...
```

Comment:
Expected type is list of lists. Should different typing be used in this way?
https://github.com/opencv/opencv/blob/5c090b9eec0cfe624fb9c3d11e18d9d1b4ff0ead/samples/dnn/edge_detection.py#L35

Reply:
Yep its typo, thanks